src/Engine/Publisher.php

```diff
@@ -38,11 +38,25 @@ public function apply_config( $config ) {
             @[
                 'control_uri' => $control_uri,
                 'control_iss' => $control_iss,
-                'key' => $key
+                'key' => $key,
+                'verify_key' => $verify_key,
+                'verify_iss' => $verify_iss
             ] = $config_entry;
             $client = new PublisherClient( $control_uri );
             if( !empty($control_iss) ) {
+                if( empty( $key ) ) {
+                    $key = '';
+                }
                 $client->set_auth_jwt( [ 'iss' => $control_iss, ], $key );
+            } else {
```

Comment:
```suggestion
            } else if ( !empty( $key ) ) {
```

If both `$control_iss` and `$key` are empty, then we shouldn't set auth at all. This is the default behavior, and would be used with local Pushpin, etc.